CoffeeChat/Controller/GoalsViewController.swift

```diff
@@ -14,14 +14,15 @@ class GoalsViewController: UIViewController {
     // MARK: - Private Data vars
     private weak var delegate: OnboardingPageDelegate?
     // TODO: change when networking with backend
-    private var goals: [String: Bool] = [
-        "Just chatting": false,
-        "Finding my people": false,
-        "Meeting someone different": false,
-        "Learning from mentors": false,
-        "Guiding mentees": false,
-        "Not sure yet": false
+    private var goals: [SimpleOnboardingCell] = [
+        SimpleOnboardingCell(name: "Just chatting", subtitle: nil),
+        SimpleOnboardingCell(name: "Finding my people", subtitle: nil),
+        SimpleOnboardingCell(name: "Meeting someone different", subtitle: nil),
+        SimpleOnboardingCell(name: "Learning from mentors", subtitle: nil),
+        SimpleOnboardingCell(name: "Guiding mentees", subtitle: nil),
+        SimpleOnboardingCell(name: "Not sure yet", subtitle: nil)
     ]
+    private var selectedGoals: [Bool] = [false, false, false, false, false, false]
```

Comment:
If we can combine `goals` and `selectedGoals` into one variable, we can avoid running into issues if one array doesn't match up